README.md

```diff
@@ -3,55 +3,124 @@ Karhunen Loève decomposed Gaussian processes with forward variable
 ```
-model.converage3(Normalized Test Inputs, Test Data, draws, plots)
+model.converage3(model.inputs, model.data, plot='sorted', bounds=1, legend=1)
 ```
-Cover
+Also insightful is the RMSE of the model's fit, which is the third positional output of 'coverage3'.
 
-## Integration
-FoKL can be used to model state derivatives and thus contains an integration method of these states using an RK4. Due to each state being modeled independently the same functionality cannot be used. The model.fits outputs should be returned to your workspace via:
+As a side note, the following attributes were added to your FoKL class 'model' after calling 'fit' which may be useful during user post-processing:
+```
+model.inputs         == all normalized inputs w/o outliers (i.e., model.traininputs plus model.testinputs)
+model.data           == all data w/o outliers (i.e., model.traindata plus model.testdata)
+
+model.betas          == betas
+model.mtx            == mtx
+model.evs            == evs
+
+model.rawinputs      == all normalized inputs w/ outliers == user's 'inputs' but normalized and formatted
+model.rawdata        == all data w/ outliers              == user's 'data' but formatted
+model.traininputs    == train set of model.inputs
+model.traindata      == train set of model.data
+model.testinputs     == test set of model.inputs
+model.testdata       == test set of model.data
+model.normalize      == [min, max] factors used to normalize user's 'inputs' to 0-1 scale of model.rawinputs
+model.outliers       == indices removed from model.rawinputs and model.rawdata as outliers
+model.trainlog       == indices of model.inputs used for model.traininputs
+model.testlog        == indices of model.data used for model.traindata
 
+model.inputs_np      == model.inputs as a numpy array of timestamps x input variables
+model.rawinputs_np   == model.rawinputs as a numpy array of timestamps x input variables
+model.traininputs_np == model.traininputs as a numpy array of timestamps x input variables
+model.testinputs_np  == model.testinputs as a numpy array of timestamps x input variables
 ```
-betas, mtx, evs = model.fit(inputs, data)
+To remove all of the above attributes so that only the hyperparameters remain, most importantly so that 'betas' does not influence the training of a new model, use:
+```
+model.clear()
 ```
-and then used as 
 
+## Integration
+FoKL can be used to model state derivatives and thus contains an integration method of these states using an RK4. Due to each state being modeled independently, the same functionality cannot be used. For the case of two states with the same inputs:
 ```
-T,Y = FoKLRoutines.FoKL.GP_Integrate([np.mean(betas1,axis=0),np.mean(betas2,axis=0)], [mtx1,mtx2], utest, norms, phis, start, stop, ic, stepsize, used_inputs)
+data = [state1, state2]
+betas = []
+mtx = []
+for i in range(2):
```

Comment:
Line 106 - IMO simpler is better sometimes for readability, is it necessary to do a for loop for just a two state model? Just might be easier to show fitting two separate models and putting them both in.

Reply:
You could then cal in gp integrate (model1.betas) as opposed to having to redefine it as a global variable

Reply:
i could see pros/cons for both approaches --- imo it'd be easier for a user to take the for loop (even if it's only 2 states here) and generalize it to many states or to a sweep of hypers, but not the other way around

i figure model1 and model2 would be obvious without an example (or at least all the tools are available from "Setup" section), but a loop might not be as obvious so this is kinda acting as a two-birds one-stone for showing loops

BUT, maybe an extra section after "Integration" called "Sweeps" or something like that? That way we keep readability (by using model1, model2) but also have docs for how to loop

(also ... forgot to add "model.clear()" at the end of the loop which is the main piece of syntax to convey to user about using loops)